<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="D90D3CQ1" name="com.scripts.EDD.TC14245_EnsureUserCanAddAnAttachementToEDDRecord" tests="1" failures="0" timestamp="25 Jun 2018 06:27:57 GMT" time="35.335" errors="1">
  <testcase name="tc14252_AddNextStepToEDDRecord" time="35.335" classname="com.scripts.EDD.TC14245_EnsureUserCanAddAnAttachementToEDDRecord">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:213)
	at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:102)
	at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
	at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
	at com.generic.BaseTest.initializeWebEnvironment(BaseTest.java:81)
	at com.scripts.EDD.TC14245_EnsureUserCanAddAnAttachementToEDDRecord.setEnviorment(TC14245_EnsureUserCanAddAnAttachementToEDDRecord.java:29)
	at com.scripts.EDD.TC14245_EnsureUserCanAddAnAttachementToEDDRecord.tc14252_AddNextStepToEDDRecord(TC14245_EnsureUserCanAddAnAttachementToEDDRecord.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- tc14252_AddNextStepToEDDRecord -->
</testsuite> <!-- com.scripts.EDD.TC14245_EnsureUserCanAddAnAttachementToEDDRecord -->
